src/react/src/components/AddLocationData.jsx

```diff
@@ -0,0 +1,336 @@
+            position: 'absolute',
+            right: '20px',
+            top: '50%',
+            transform: 'translateY(-50%)',
+        }),
+    });
+
+function AddLocationData({ classes, userHasSignedIn, fetchingSessionSignIn }) {
+    if (fetchingSessionSignIn) {
+        return (
+            <AppGrid title="Contribute">
+                <Grid container className="margin-bottom-64">
+                    <Grid item xs={12}>
+                        <CircularProgress />
+                    </Grid>
+                </Grid>
+            </AppGrid>
+        );
+    }
+
+    if (!userHasSignedIn) {
+        return (
+            <AppGrid title="Contribute">
+                <Grid container className="margin-bottom-64">
+                    <Grid item xs={12}>
+                        <Link to={authLoginFormRoute} href={authLoginFormRoute}>
+                            Log in to contribute to Open Supply Hub
+                        </Link>
+                    </Grid>
+                </Grid>
+            </AppGrid>
+        );
+    }
+
+    return (
+        <div className={classes.container}>
+            <Typography variant="display3" className={classes.title}>
+                Add production location data to OS Hub
+            </Typography>
+            <Typography variant="subheading" className={classes.description}>
+                Contribute your data here to help build the world’s most
+                complete, open and accessible map of global production:
+            </Typography>
+            <div className={classes.dataOptions}>
+                <Paper className={classes.card}>
+                    <div className={classes.cardRectangleView}>
+                        <RectangleCardFigure />
+                    </div>
+                    <div>
+                        <PlaylistIcon className={classes.cardIcon} />
+                        <Typography
+                            variant="display"
+                            className={classes.cardTitle}
+                        >
+                            Upload a dataset with multiple production locations
+                            using a{' '}
+                            <span className={classes.highlight}>
+                                spreadsheet.
+                            </span>
+                        </Typography>
+                        <Typography
+                            variant="subheading"
+                            className={classes.cardSub}
+                        >
+                            This option is best if you have a large number of
+                            production locations to contribute.
+                        </Typography>
+                        <Button
+                            color="secondary"
+                            variant="text"
+                            component={Link}
+                            className={classes.buttonStyle}
+                            to="/contribute/multiple-locations"
```

Comment:
Can you move the link to the constants?jsx such a way as multipleLocationRoute var?

Reply:
Done.